CGATPipelines/Pipeline/Execution.py

```diff
@@ -367,11 +367,25 @@ def run(**kwargs):
     options.update(getCallerLocals().items())
     options.update(kwargs.items())
 
-    # insert a few legacy synonyms
-    options['cluster_options'] = options.get('job_options',
-                                             options['cluster_options'])
```

Comment:
@IanSudbery , no need to worry about backwards compatibility as long as the new behaviour is better and saner.